packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -46,70 +54,57 @@ The [Brands API](/docs/reference/api/brands/) is a feature (currently in Early A
+* When multiple translations have been added for a template, the translation provided in the default language appears at the top of the list. You can designate any added translation as the default language by selecting it from the **Default Language** drop-down box. Doing so reorders the list of added translations automatically. You can edit the templates through the pencil icon, but you can't delete the default language template.
 
-## Enhanced Email Macros
+1. In the Admin Console, go to **Customizations** > **Emails**.
+1. Choose an email template that you've customized. The default language version appears in the **Custom Email** table.
+1. Click **Add Translation**, and then select a language from the drop-down box. If the **Add Translation** button isn't available, this template hasn't been customized. See [Edit a default email template](#edit-a-default-email-template]).
+1. Make your translated edits, and then click **Add Translation**.
+1. Repease steps 3 and 4 for additional languages.
 
-<ApiLifecycle access="ea" />
+To delete all custom translations and revert to the Okta template, click **Reset to Default**.
 
-> **Note:** To access this self-service EA feature, you need to enable **Enhanced Email Macros** in the Feature Manager. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features). For free orgs, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867) to request email customization.
+> **Note:** It may be more convenient to copy and paste the HTML from the message body into a text editor, compose your custom translation, then copy and paste it back into the message body.
 
-> **Note:** The EA Enhanced Email Macros feature is supported in Okta Identity Engine. <ApiLifecycle access="ie" />
+### Velocity Templating Language
 
-The Enhanced Email Macros feature provides you with the functionality to customize the email templates with even greater flexibility than Expression Language. It changes the underlying email templating engine from Expression Language variables to [Velocity Templating Language (VTL)](https://velocity.apache.org/). This allows you to customize your org's email templates so that you can use:
+[Velocity Templating Language (VTL)](https://velocity.apache.org/engine/1.7/user-guide.html) allows you to customize your org's email templates so that you can use:
 
 - enhanced conditional logic
 - all of the attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object)
 - some of the org attributes in these macros
```

Comment:
should we just list all the attributes instead of this unhelpful set of bullets? (which I wrote)

Graham just updated this doc: https://help.okta.com/en/oie/Content/Topics/Settings/velocity-variables.htm